inlineplz/interfaces/github.py

```diff
@@ -12,11 +12,17 @@
 
 class GitHubInterface(InterfaceBase):
     def __init__(self, owner, repo, pr, token, url=None):
+        self.github = None
+        # TODO: support non-PR runs
+        try:
+            pr = int(pr)
+        except ValueError:
+            return
         if not url:
-            self.gh = github3.GitHub(token=token)
+            self.github = github3.GitHub(token=token)
         else:
-            self.gh = github3.GitHubEnterprise(url, token=token)
-        self.pull_request = self.gh.pull_request(owner, repo, pr)
+            self.github = github3.GitHubEnterprise(url, token=token)
```

Comment:
`pylint: Redefinition of self.github type from github3.github.GitHub to github3.github.GitHubEnterprise (redefined-variable-type)`